lib/web_ui/test/window_test.dart

```diff
@@ -41,6 +41,12 @@ void testMain() {
     await window.resetHistory();
   });
 
+  test('EnginePlatformDispatcher.instance.implicitView should be non-null', () async {
```

Comment:
Same.

Reply:
> All other platforms - mobile and web - will have an implicit view instead. In other words, this test will be correct for the foreseeable future.

I think we'd rather not assume that (and we don't have to, right?). The user might very possibly attach multiple views from the native code, and the framework will handle them. And with that, the user might even choose not to use the implicit view.

Reply:
> All other platforms - mobile and web - will have an implicit view instead.

Web will (possibly) want to render to multiple different target DIVs at some point, so I guess we're going to have multiple "views" too?

Reply:
> 



> > All other platforms - mobile and web - will have an implicit view instead.
> 
> Web will (possibly) want to render to multiple different target DIVs at some point, so I guess we're going to have multiple "views" too?

To clarify there's two semi-related things here:

1. Multi-view: all platforms, including web, will have multi-view support
2. Implicit view: this replaces the `window` global

Having an implicit view does not block multi-view. It's just the replacement for the `window` global.

Reply:
Thanks for the clarification @loic-sharma (expect a quick call from me soon-ish though, I want to start redoing stuff in the web engine so the multi-view lands more easily!)